lib/msal-browser/docs/resources-and-scopes.md

```diff
@@ -77,7 +78,7 @@ Consider a slightly different case:
   msalInstance.loginPopup(loginRequest);
   // will fail and fallback to an interactive method prompting a consent screen
   msalInstance.acquireTokenSilent(tokenRequest);
-  // will succeed and return an Access Token with scopes "openid", "profile", "User.Read" and "api://<myCustomApiClientId>/My.Scope"
+  // will succeed and return an Access Token with scope "api://<myCustomApiClientId>/My.Scope"
```

Comment:
Don't we return `openid`, `profile` as default scopes?

Reply:
@sameerag can't quite remember how to behavior was when this was written, but not anymore (makes sense if they are considered Graph scopes, hence different resource)

Reply:
`openid` and `profile` are OIDC scopes, not Graph scopes. It just so happens that AAD returns `openid` and `profile` with Graph requests, but not for other resources. I don't remember the reason why this is the case.